api/db/postgres.go

```diff
@@ -88,30 +61,9 @@ func (pR *PostgresRequests) FindOneDBSecurityTest(
 func (pR *PostgresRequests) FindOneDBAnalysis(
 	mapParams map[string]interface{}) (types.Analysis, error) {
 	analysisResponse := []types.Analysis{}
-	analysis, ok := mapParams["RID"]
-	if !ok {
-		return types.Analysis{}, errors.New("Could not find RID field")
-	}
-	analysisQuery := `SELECT
-					RID,
-					repositoryURL,
-					repositoryBranch,
-					commitAuthors,
-					status,
-					result,
-					errorFound,
-					containers,
-					startedAt,
-					finishedAt,
-					codes,
-					huskyciresults
-				FROM
-					analysis
-				WHERE
-					RID = $1`
-
+	query, params := ConfigureQuery(`SELECT * FROM "analysis"`, mapParams)
 	if err := pR.DataRetriever.RetrieveFromDB(
-		analysisQuery, &analysisResponse, analysis); err != nil {
+		query, &analysisResponse, []string{"commitAuthors"}, params...); err != nil {
```

Comment:
Why is this `[]string{"commitAuthors"}` needed here? 🙃

Reply:
@rafaveira3 I know that this field is not in use right now. But, this is not a problem since no data is sent in this field, its content will not be converted to the valid data format to be interpreted by the psql plugin. However, I can't discard the usage of this field for the future since its column exists in the table and it needs to be in an array format. If this field is not required anymore it needs to be removed in the future either in code and in DB.